doc/test_messages_documentation.py

```diff
@@ -108,6 +125,10 @@ def _runTest(self) -> None:
         self._linter.check([str(self._test_file[1])])
         expected_messages = self._get_expected()
         actual_messages = self._get_actual()
+        if self.is_good_test_file():
+            assert actual_messages.total() == 0  # type: ignore[attr-defined]
```

Comment:
I think it is because `MessageCounter` is still `Counter` without the `totals`. But this is fine for me!